plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/model/RepositoryReference.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*global Ext, NX*/
+
+/**
+ * Repository reference model.
+ *
+ * @since 3.0
+ */
+Ext.define('NX.coreui.model.RepositoryReference', {
+  extend: 'Ext.data.Model',
+  fields: [
+    'id',
+    'name',
+    'type',
+    'format'
```

Comment:
maybe we should pick up https://issues.sonatype.org/browse/NEXUS-7898 soon, to avoid any more copy-paste degradation?